src/python/pants/backend/python/tasks/setup_py.py

```diff
@@ -300,7 +300,7 @@ def collect_reduced_dependencies(current):
 class SetupPy(Task):
   """Generate setup.py-based Python projects."""
 
-  SOURCE_ROOT = b'src'
+  SOURCE_ROOT = 'src' if PY3 else b'src'
```

Comment:
This file is a little hacky to get to work on both versions. 

It relies on `pprint` to generate the content written to file. In Py2, unicode is prefaced with `u`, and in Py3 bytes are prefaced with `b`. The goal is to have no prefix displayed, so to do this we have to use bytes in Py2 and unicode in Py3.